### PR TITLE
Various documentation updates and fixes

### PR DESCRIPTION
**Pull request by JaimeMagiera**
Updates to the FAQ, Known Issues, README.

### CODE REVIEW

**Comment by JaimeMagiera** [comment]
Rebased.

File: FAQ.md [hunk 1]
```diff
@@ -121,7 +121,11 @@ accessible and user workloads would be evicted and rescheduled as nodes are upda
 
 ## Should I use Fedora CoreOS 32 or 33 for installing on User Provisioned Infrastructure?
 
+<<<<<<< HEAD
 There is currently an issue with using Fedora CoreOS 33 for UPI installations. Please use the FCOS 32 image. After installation, the nodes will have rebooted and updated to the latest stable FCOS 33. 
+=======
+There is currently an issue with using Fedora CoreOS 33 for UPI installations. Please use the FCOS 32 image. After installation, the nodes will have rebooted and updated to the latest stable FCOS 33. For example, you can use the [FCOS 32 .ova image](https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/32.20201104.3.0/x86_64/fedora-coreos-32.20201104.3.0-vmware.x86_64.ova) for vSphere installations. 
+>>>>>>> 0cbed72... Various documentation updates and fixes
```
Comment (sandrobonazzola) [inline FAQ.md:128]: Please fix this rebase conflict

File: KNOWN_ISSUES.md [hunk 2]
```diff
@@ -1,10 +1,24 @@
...
+  **Workaround**: None available at this time. 
 
-## [Invalid signature store used](https://github.com/openshift/okd/issues/426)
-  Upgrades to latest stable version require `--force` when `oc adm upgrade` call
+## [Invalid signature error when attempting cluster updates](https://github.com/openshift/okd/issues/426)
+  **Effected Versions:** 4.6.0-0.okd-2020-11-27-200126, 4.6.0-0.okd-2020-12-12-135354, 4.6.0-0.okd-2021-01-17-185703  
+  **Description:** Cluster upgrades will fail due to incorrect image signatures.   
+  **Workaround:** Perform the upgrade in the terminal using the --force flag  
+  ```bash
+  oc adm upgrade --to-latest=true --force=true
```
Comment (sandrobonazzola) [inline KNOWN_ISSUES.md:13]: If you already tried to upgrade above command won't be enough.
Output will be like:
```
oc adm upgrade --to-latest=true --force=true
error: already upgrading.

  Reason: ImageVerificationFailed
  Message: Unable to apply 4.6.0-0.okd-2021-01-23-132511: the image may not be safe to use



If you want to upgrade anyway, use --allow-upgrade-with-warnings.
```

Comment (sandrobonazzola) [reply KNOWN_ISSUES.md:13]: And then:

```
# oc adm upgrade --to-latest=true --force=true --allow-upgrade-with-warnings
warning: --allow-upgrade-with-warnings is bypassing: already upgrading.

  Reason: ImageVerificationFailed
  Message: Unable to apply 4.6.0-0.okd-2021-01-23-132511: the image may not be safe to use

warning: --force overrides cluster verification of your supplied release image and waives any update precondition failures.
Updating to latest version 4.6.0-0.okd-2021-01-23-132511
```

**Review by sandrobonazzola — CHANGES_REQUESTED** [review]
See inline comments

File: KNOWN_ISSUES.md (hunk 2 above)
Comment (vrutkovs) [reply KNOWN_ISSUES.md:13]: Right, we should add `oc adm upgrade --reset && oc adm update ..` instead

File: FAQ.md (hunk 1 above)
Comment (JaimeMagiera) [reply FAQ.md]: Sorry, fixed.

File: KNOWN_ISSUES.md (hunk 2 above)
Comment (JaimeMagiera) [reply KNOWN_ISSUES.md]: Added --reset before forced upgrade and use of --allow-upgrade-with-warnings for forced upgrade.

**Review by sandrobonazzola — APPROVED** [review]
/lgtm

File: FAQ.md [hunk 3]
```diff
@@ -121,8 +121,7 @@ accessible and user workloads would be evicted and rescheduled as nodes are upda
 
 ## Should I use Fedora CoreOS 32 or 33 for installing on User Provisioned Infrastructure?
 
-There is currently an issue with using Fedora CoreOS 33 for UPI installations. Please use the FCOS 32 image. After installation, the nodes will have rebooted and updated to the latest stable FCOS 33. 
-
+There is currently an issue with using Fedora CoreOS 33 for UPI installations. Please use the FCOS 32 image. After installation, the nodes will have rebooted and updated to the latest stable FCOS 33. For example, you can use the [FCOS 32 .ova image](https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/32.20201104.3.0/x86_64/fedora-coreos-32.20201104.3.0-vmware.x86_64.ova) for vSphere installations.
```
Comment (vrutkovs) [inline FAQ.md:124]: I don't think its still actual. F33 as a base should work fine.

Lets remove this and then re-add if folks complain?

Comment (JaimeMagiera) [reply FAQ.md]: Done.

**Review by vrutkovs — APPROVED** [review]
Thanks!
/lgtm

**Pull request merged**